doc/source/install/docker.rst

```diff
@@ -15,74 +15,109 @@ provided via ``apt`` and ``yum`` may be outdated and cause errors.
 
 .. note::
 
-   By default, Docker pulls the image from Docker Hub if it is not
+   By default, Docker pulls the image from Docker Hub if the image is not
    available locally.
 
-   The image only contains the most essential binaries for |PXC| to
-   run. This means that some utilities included into a |PS| or |MySQL|
+   The image contains only the most essential binaries for |PXC| to
+   run. Some utilities included in a |PS| or |MySQL|
    installation might be missing from the |PXC| Docker image.
 
 The following procedure describes how to set up a simple 3-node cluster
-for evaluation and testing purposes,
-with all nodes running |PXC| 8.0 in separate containers on one host:
+for evaluation and testing purposes. Do not use these instructions in a
+production environment. For a production environment, you should generate and
```

Comment:
Maybe add somewhere that certificates generated by mysql init process are self signed and that's the reason why not use them in production - just so that reasoning is clear.

Reply:
Will add.